docs/examples/frameworks/jax/flax-basic_example.ipynb

```diff
@@ -16,7 +16,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #21.        images = fn.reshape(images, shape=[image_size * image_size])
</pre><p>How about just:</p><pre class="ql-syntax" spellcheck="false">Line #21.        images = fn.reshape(images, shape=[-1])
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5004/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I did this sort of on purpose to clearly show what will be the shape of the returned tensor. I spend some time debugging how to match DALI return value with the model input. I am open to changing it somehow but I would like to maintain this clarity since this is for educational purposes. Do you have some ideas how we could do it?</p><p> </p>

Reply:
<p>I did something. Please, take a look if it works better. </p>

Reply:
Looks good.